scripts/lib/setup_path_on_import.py

```diff
@@ -3,11 +3,10 @@
 Also add Zulip's root directory to sys.path
 """
 
-import os
-from os.path import dirname, abspath
+import os.path
```

Comment:
Only `import os`?

Reply:
When the file uses other modules like ```os.environ``` I keep ```import os```. What do you think?

Reply:
I see, I guess we'll have to wait for someone else to look at this.